.github/workflows/rust.yml

```diff
@@ -1,22 +1,38 @@
 name: Rust
 
-on:
+on: 
```

Comment:
delete space please - what linter/formatter/IDE you using ?

Reply:
I am using VSCode without a linter or formatter for rust besides the builtin one. Any that you recommend. Fixed in commit [5a3019](https://github.com/Chleba/netscanner/pull/33/commits/5a301395f3004340d61affffcb9fb3c3f8d3cb66)

Reply:
Well I cannot recommend anything else then neovim with rust tools plugins installed. Sorry :) I'm just too used to it right now to switch to anything different.